package.json

```diff
@@ -20,7 +20,7 @@
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
     "@bcrs-shared-components/contact-info": "1.2.4",
     "@bcrs-shared-components/corp-type-module": "1.0.9",
-    "@bcrs-shared-components/court-order-poa": "2.1.4",
+    "@bcrs-shared-components/court-order-poa": "3.0.0",
```

Comment:
Since you're checking in new code, you should also update the app version.

Reply:
Shoot, I forgot to update the version.  Adding now ...